packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -79,424 +130,549 @@
     </dd>
     <dt>icon <code>string</code></dt>
     <dd>
-      <p>Any valid flight-icon name. If none is defined,
-        <span class="dummy-code default">user</span>
-        is set by default</p>
+      <p>Acceptable value: any Flight icon name.</p>
+      <p>Default: <span class="default">"user"</span></p>
     </dd>
     <dt>hasChevron <code>boolean</code></dt>
     <dd>
-      <p>
-        Acceptable values:
-      </p>
-      <ol>
-        <li class="default">true</li>
-        <li>false</li>
-      </ol>
+      <p>Default: <span class="default">true</span></p>
       <p>Per design,
-        <span class="dummy-code">false</span>
+        <code class="dummy-code">false</code>
         is only acceptable when the "more-horizontal" icon is used; as such, it is set to
-        <span class="dummy-code">true</span>
+        <code class="dummy-code">true</code>
         by default.</p>
     </dd>
-    <dt>
-      imageSrc
-      <code>string</code>
-    </dt>
+    <dt>imageSrc <code>string</code></dt>
     <dd>
       The URL of an image to be used instead of an icon (e.g., as an avatar).
     </dd>
+    <dt>“splattributes”</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
   </dl>
-  <hr />
-  <p class="dummy-paragraph" id="listitem-api">
-    This is the API for the
-    <span class="dummy-code">Dropdown::ListItem</span>
-    component:
-  </p>
-  <dl class="dummy-component-props" aria-labelledby="listitem-api">
-    <dt>
-      item
-      <code>
-        enum
-      </code>
-    </dt>
+
+  <h4 class="dummy-h4">ListItem::CopyItem</h4>
+  {{! TODO: add props here if we decide to use the current version }}
+  <p class="dummy-paragraph">TODO??</p>
+
+  <h4 class="dummy-h4">ListItem::Description</h4>
+  <p class="dummy-paragraph">Here is the API for the "description" list item component (yielded as
+    <code class="dummy-code">Description</code>):</p>
+  <dl class="dummy-component-props" aria-labelledby="listitem-description-api">
+    <dt>text <code>string</code></dt>
     <dd>
-      <p>
-        Acceptable values:
-      </p>
-      <ol>
-        <li class="default">interactive</li>
-        <li>copy-item</li>
-        <li>description</li>
-        <li>generic</li>
-        <li>separator</li>
-        <li>title</li>
-      </ol>
-      <p>🚨 Important Note: the
-        <span class="dummy-code">generic</span>
-        is a yield and the developer is completely responsible for any item yielded there (including the accessibility
-        of that item). This is provided as an escape hatch to empower developers to more easily transition to the design
-        system. It should be considered as a temporary measure with plans for replacement.</p>
+      <p>The text to be used for the description.</p>
+      <p><em>If no text value is defined an error will be thrown.</em></p>
+    </dd>
+    <dt>“splattributes”</dt>
```

Comment:
```suggestion
    <dt>...attributes</dt>
```

[issue] We use the technically accurate names elsewhere, it should be used here as well.